cosmos/nibiru-devnet-2.json

```diff
@@ -0,0 +1,51 @@
+{
+  "chainId": "nibiru-devnet-2",
```

Comment:
We can't handle nibiru-devnet-1, nibiru-devnet-2 as different chains.
In turn, you should register only one chain information per a chain identifier. If you're confused about the term, could you please check the explanation about chain identifier on README one more time?

Reply:
https://github.com/chainapsis/keplr-chain-registry?tab=readme-ov-file#chain-registration-directory-structure